db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv

```diff
@@ -0,0 +1,27 @@
+code,descr,lang_code,is_active,cr_by,cr_dtimes
```

Comment:
For this jira Id, why do we need these changes ?

Reply:
Rows to be added for   
WORKFLOW_PAUSE
WORKFLOW_RESUME

Reply:
Then that case the file should contain only 2 new lines for those 2 types rt ?